config/_default/menus.toml

```
@@ -4,7 +4,7 @@
 #   The weight parameter defines the order that the links will appear in.
 
 [[main]]
-  name = "2i2c Hubs"
+  name = "2i2c managed JupyterHubs"
```

```suggestion
  name = "Managed JupyterHubs"
```

I think we should just remove the name "2i2c" from here otherwise it's going to take a lot of navbar space, and I think it's implicit